ChangeLog

```diff
@@ -1,3 +1,5 @@
+1.11.0
+
 Remove automigrate spassword inside rpm
```

Comment:
I'd suggest to remove empty blank lines to "compact" the changelog in this version. Similar to 1.5.4 below.

Reply:
fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/166/commits/e1d85584bab5b788a3671c173b3a06f845b9d413